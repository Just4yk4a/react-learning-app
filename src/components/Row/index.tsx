import * as React from 'react';

interface IProps {
    car: any;
}

export default class Row extends React.Component<IProps, {}> {

    public render() {
        const car = this.props.car;
        return (
            <tr>
                <td>{car.vin}</td>
                <td>{car.brand}</td>
                <td>{car.year}</td>
                <td>{car.color}</td>
            </tr>
        );
    }
}