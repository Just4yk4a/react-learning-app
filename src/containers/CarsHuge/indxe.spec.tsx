import {shallow} from "enzyme";
import * as React from "react";
import Row from "./row";

describe("Test Row component", () => {
    it("Row", () => {
        const prop:any = {
            brand: "VOLVO",
            color: "RED",
            vin: 123,
            year: 1990
            };

        const output = shallow(<Row car={prop}/>);
        expect(output).toMatchSnapshot();
    });
});