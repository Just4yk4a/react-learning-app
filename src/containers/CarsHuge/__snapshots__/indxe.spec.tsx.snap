// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Row component Row 1`] = `
<tr>
  <td>
    123
  </td>
  <td>
    VOLVO
  </td>
  <td>
    1990
  </td>
  <td>
    RED
  </td>
</tr>
`;
